packages/ses/src/evaluate.js

```diff
@@ -1,7 +1,7 @@
 // Portions adapted from V8 - Copyright 2016 the V8 project authors.
 // https://github.com/v8/v8/blob/master/src/builtins/builtins-function.cc
 
-import { throwTantrum } from './assertions.js';
+import { assert, details } from '@agoric/assert';
```

Comment:
Perhaps it would make sense to export `assert`, `details`, and `quote`, then leave it to the consumer to import them by convention like:

```js
import { assert, details as d, quote as q } from '@agoric/assert';
```

This might be a good way to balance brevity and put the explanation as close to the confused reviewer as possible.

Reply:
Done. Something very nice about this coupled use of `d` and `q` is that, at least in the font I'm using, they are vertical mirror images of each other.